src/_guides/language/language-tour.md

```diff
@@ -56,6 +56,19 @@ This content has moved to [Built-in types](/language/built-in-types#booleans).
 
 This content has moved to [Collection types](/language/collection-types#lists).
 
+<a id="trailing-comma"></a>
```

Comment:
Should these be added at the end of the page as an archived section of similar?

Reply:
Everything on the page is archived. I'm trying to keep it near their old location, like others on the page were, these ones were just forgotten.

Reply:
Sgtm